accounts/forms.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+from django import forms
+from django.contrib.auth import get_user_model
+
+
+def split_full_name(full_name):
+    if not full_name:
+        first_name, last_name = None, None
+    else:
```

Comment:
I think this, this first level if else is redundant, try to refactor your function with fewer if else statements to make it more clean and readable.

HInt: set firstname and last name as Non in the start of the function and then use if/else checks to update require fields.

Reply:
done